Library/Formula/mitie.rb

```diff
@@ -14,27 +14,34 @@ class Mitie < Formula
 
   depends_on :python if MacOS.version <= :snow_leopard
 
+  option "with-models", "Install v0.2 models (~415MB)"
```

Comment:
Their readme just says [you need the models in order to run the provided examples](https://github.com/mit-nlp/MITIE#initial-setup).